src/ui.rs

```diff
@@ -576,7 +576,103 @@ impl<'a> UiFrame<'a> {
+                    liability disclaimers.\n\
+                    \n\
+                    If you register for a free trial of the H.U.R.B.A.N. selector software, this EULA \
+                    agreement will also govern that trial. By clicking 'accept' or installing and/or \
+                    using the H.U.R.B.A.N. selector software, you are confirming your acceptance of \
+                    the Software and agreeing to become bound by the terms of this EULA agreement.\n\
+                    \n\
+                    If you are entering into this EULA agreement on behalf of a company or other legal \
+                    entity, you represent that you have the authority to bind such entity and its \
+                    affiliates to these terms and conditions. If you do not have such authority or if \
+                    you do not agree with the terms and conditions of this EULA agreement, do not \
+                    install or use the Software, and you must not accept this EULA agreement.\n\
+                    \n\
+                    This EULA agreement shall apply only to the Software supplied by Slovak Design \
+                    Center and Subdigital s.r.o. herewith regardless of whether other software is \
+                    referred to or described herein. The terms also apply to any Slovak Design Center \
+                    and Subdigital s.r.o. updates, supplements, Internet-based services, and support \
+                    services for the Software, unless other terms accompany those items on delivery. \
+                    If so, those terms apply. This EULA was created by EULA Template for H.U.R.B.A.N. \
+                    selector."));
+                ui.new_line();
+                ui.text(imgui::im_str!("LICENSE AGREEMENT"));
+                ui.new_line();
+                ui.text_wrapped(imgui::im_str!("\
+                    Slovak Design Center and Subdigital s.r.o. hereby grants you a personal, \
+                    non-transferable, non-exclusive licence to use the H.U.R.B.A.N. selector software \
+                    on your devices in accordance with the terms of this EULA agreement.\n\
+                    \n\
+                    You are permitted to load the H.U.R.B.A.N. selector software (for example a PC, \
+                    laptop, mobile or tablet) under your control. You are responsible for ensuring \
+                    your device meets the minimum requirements of the H.U.R.B.A.N. selector software.\n\
+                    \n\
+                    You are not permitted to:\n\
+                    * Edit, alter, modify, adapt, translate or otherwise change the whole or any part of \
+                    the Software nor permit the whole or any part of the Software to be combined with or \
+                    become incorporated in any other software, nor decompile, disassemble or reverse \
+                    engineer the Software or attempt to do any such things\n\
```

Comment:
Not sure this clause makes sense. To me it looks to be in direct violation with the GNU GPL. People can modify and adapt the software. We already provide the source they can build from, so no need to decompile anything. Is this clause necessary?

If we modify this clause, let's not forget to modify the copy of it in `eula.md`

Reply:
okay, i'll remove it